pkg/kubectl/resource/helper.go

```diff
@@ -72,6 +73,15 @@ func (m *Helper) List(namespace, apiVersion string, export bool, options *metav1
 		// TODO: I should be part of ListOptions
 		req.Param("export", strconv.FormatBool(export))
 	}
+
+	if requestTable {
+		table := &metav1alpha1.Table{}
+		err := req.
+			AsTable(metav1.GroupVersion{}).
+			Do().Into(table)
```

Comment:
@deads2k I'm thinking it'd be best to just have a codec that handles this, however I'm not too familiar with this part of the codebase: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/runtime/serializer/negotiated_codec.go#L34